client/my-sites/checkout/upsell-nudge/professional-email-upsell/index.tsx

```diff
@@ -68,12 +72,28 @@ const ProfessionalEmailUpsell = ( {
 
 	const optionalMailboxFields = [ 'alternativeEmail', 'name' ];
 
-	const formattedPrice = translate( '{{price/}} /mailbox /month', {
-		components: {
-			price: <span>{ formatCurrency( productCost ?? 0, currencyCode ) }</span>,
-		},
-		comment: '{{price/}} is the formatted price, e.g. $20',
-	} );
+	const resolveFormattedPrice = (
```

Comment:
This relates to my previous comment, let's avoid the `resolve` term:

```suggestion
	const getFormattedPrice = (
```

Reply:
Note we could actually move that function out from `ProfessionalEmailUpsell`, which would help with making the former shorter.

Reply:
I will iterate on this in a next pull request, adding a billing interval. If that works for you, I'll defer this change to the next PR.